.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
are you sure you want to commit your vscode config? it is likely to conflict with configs of everyone who also uses vscode